omnibus/files/private-chef-ctl-commands/install.rb

```diff
@@ -31,6 +24,7 @@
     exit 1
   end
 
+  attributes_path = "#{base_path}/embedded/cookbooks/install_params.json"
```

Comment:
Just for consistency I'd add a chef_args var here and pass it in, since the same is done in the cleanup command, but that's more stylistic for consistency than functionality.